development/ts-migration-dashboard/common/build-module-partitions.ts

```diff
@@ -366,13 +431,24 @@ function partitionModulesByLevel(
     modulePartitions[i] = { level: i + 1, children: [] };
   }
   Object.values(modulesById).forEach((module) => {
+    const hasBeenConvertedToTypeScript =
```

Comment:
These changes help us to know how to style a box on the dashboard depending on whether a file was converted from JS to TS, or whether it was originally written in TS.